package.json

```diff
@@ -38,7 +38,8 @@
   "defaults": {
     "roslyn": "4.8.0-1.23367.7",
     "omniSharp": "1.39.7",
-    "razor": "7.0.0-preview.23301.2"
+    "razor": "7.0.0-preview.23328.2",
```

Comment:
I updated Razor to the original version from @davidwengier's PR: https://github.com/dotnet/vscode-csharp/pull/5879/commits/dd29e3c4f5c46c5f0ea59822082459b3593a56f0